interfaces/builtin/optical_drive.go

```diff
@@ -19,13 +19,27 @@
 
 package builtin
 
-const opticalDriveSummary = `allows read access to optical drives`
+import (
+	"fmt"
+
+	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/interfaces/apparmor"
+	"github.com/snapcore/snapd/snap"
+)
+
+const opticalDriveSummary = `allows read by default and optionally write access to optical drives`
```

Comment:
"allows access to optical drives"

This should be enough. Details about inner workings can go into the docs.